iib/workers/api_utils.py

```diff
@@ -5,13 +5,14 @@
 import requests
 from requests.packages.urllib3.util.retry import Retry
 import requests_kerberos
+from typing import Any, Dict, Optional
```

Comment:
looks like `typing` is a python standard library. This line should go after line 3 and before the empty line

Reply:
It should be okay in every file now.